File: C/PRAK603-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int main () {
    int x, y, i, hasil;
    scanf ("%d %d", &x, &y);
    if (x != y) {
        printf("Jumlah tidak sama");
        return 0;}
    int bilangan[x];
    int bilangan2[y];
    for (i=0;i<x;i++) {
        scanf("%d", &bilangan[i]); }
    for (i=0;i<y;i++) {
        scanf("%d", &bilangan2[i]); }
    for (i=0;i<x;i++) {
        hasil=bilangan[i] * bilangan2[i];
        printf("%d ", hasil);}
    return 0;
}

File: PYTHON/PRAK603-2210817220029-SitiAinurRahmawati.py
kolom1, kolom2 = map(int, input().split())
if kolom1 != kolom2 :
    print("Jumlah tidak sama")
else :
    baris1 = list(map(int, input().split()))
    baris2 = list(map(int, input().split()))
    hasil = []
    for i in range (kolom1) :
        hasil.append(baris1[i]*baris2[i])
    for elemen in hasil :
        print(elemen, end=" ")